Gems/Terrain/Assets/Shaders/Terrain/TerrainMacroHelpers.azsli

```diff
@@ -61,13 +61,13 @@ void SampleMacroTexture(float2 worldPosition, float2 positionDdx, float2 positio
         float2 ddx_macroUv = positionDdx / macroUvSize;
         float2 ddy_macroUv = positionDdx / macroUvSize;
 
-        if (macroMaterialData.m_colorMapId != 0xFFFF)
+        if (macroMaterialData.m_colorMapId != 0xFFFFFFFF)
```

Comment:
nit: magic number, could we get a shader side define for InvalidIndex

Reply:
Hm, that's probably worth changing if AR fails, otherwise I'll do it in my next PR.